client/js/lounge.js

```diff
@@ -17,7 +17,6 @@ $(function() {
 		"/notice",
 		"/op",
 		"/part",
-		"/query",
```

Comment:
I can't recall using any client that auto-whois'd on `/query`, no. But I also don't recall using any that didn't open a query window for `/msg`, so maybe I have limited experience.